templates/shared/_pro-contact-us-form.html

```diff
@@ -378,6 +380,14 @@ <h2 class="p-heading--3 p-card__title">Need help with Ubuntu?</h2>
 
   function getCheckboxItemsAsCSV(fieldset) {
     const checkboxes = Array.from(fieldset.querySelectorAll("input[class='p-checkbox__input']"));
-    return checkboxes.filter(item => item.checked).map(item => item.value).join(", ");
+    checkboxes.filter(item => item.checked).map(item => item.value).join(", ");
+    return checkboxes
+  }
+
+  function validateCheckbox(event, fieldsetId) {
+    const checkboxes = Array.from(document.getElementById(fieldsetId).querySelectorAll("input[class='p-checkbox__input']"));
+    if (event.currentTarget.checked) {
+      checkboxes[0].removeAttribute("required");
+    }
```

Comment:
For some reason, checkboxes don't validate natively, so I had to add this. But strangely, radios do have it, so no additional JS